### DISCUSSION TITLE
About log4j2 all async log（AsyncLoggerContextSelector）

### DISCUSSION THREAD

**Discussion by gengxiangyu** [apache/skywalking#8132] [Q&A] [answered]
In this mode, the traceId of the skywalking UI related log will be empty, but in fact, there is this log, but there is no display of the traceId. How do you solve this situation, or you can't retrieve it according to the traceId

![image](https://user-images.githubusercontent.com/45058980/141760771-6c743c86-10cb-46ae-97a6-c3630fc8446c.png)

**Comment by wu-sheng** [comment, marked as answer]
You may like to read this, https://github.com/apache/skywalking/discussions/8124

**Reply by gengxiangyu** [reply]
I saw this, but I did configure it like this

![image](https://user-images.githubusercontent.com/45058980/141761607-4c5c9e82-ce69-432f-80f7-21b8b998014f.png)

**Reply by gengxiangyu** [reply]
However, it is useless and will not be displayed. I started it with system properties, this system property

-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector

**Reply by wu-sheng** [reply]
We can't afford debug for everyone. We work locally, and many other people work too. It is better you do more checks.

**Reply by gengxiangyu** [reply]
ok，thanks
